.github/build-test.yml

```diff
@@ -0,0 +1,30 @@
+name: Node CI
```

Comment:
You need to put this file into .github/workflows folder ;)

Reply:
Whoops! :sweat_smile: